### ISSUE TITLE
Is this related to SwarmCity EthereumGasStation?

### ISSUE THREAD

**Issue by mesqueeb** [EthereumGasStation/ethereumgasstationserver#1]
See https://github.com/swarmcity/SCLabs-gasstation-service

Are these related?

**Comment by sponnet** [comment]
Yes this is an implementation that resulted from the previous work on the repo you mentioned. The SC repo has been archived.
